src/services/authentication/strategies/TSPStrategy.js

```diff
@@ -0,0 +1,90 @@
+const { AuthenticationBaseStrategy } = require('@feathersjs/authentication');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@feathersjs/authentication'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)